src/heap.cpp

```diff
@@ -320,6 +348,11 @@ bool heap_command_impl(const char* args)
  */
 bool ref_command_impl(const char* args)
 {
+	if (!CA_HEAP) {
```

Comment:
Maybe a macro like `ENSURE_CA_HEAP_INITIALIZED` will save some typing and be easy to change when later we think `CA is not initialized` is too concise and revise the error message. But this is a cosmetic issue.

Reply:
updated with a macro and clearer message.